lib/docs/src/tab.stories.tsx

```diff
@@ -1,55 +1,101 @@
+		noPadding: Utils.arg("boolean", "Visual"),
+		fullHeight: Utils.arg("boolean", "Visual"),
+		style: Utils.arg(Object.keys(Tabs.styles), "Visual"),
+		children: Utils.arg(null, "Controlled"),
+		activeTab: Utils.arg(null, "Controlled"),
+		setActiveTab: Utils.arg(null, "Controlled"),
+		initialTab: Utils.arg(null, "Uncontrolled"),
+	},
+};
+
+Utils.page.component(meta, {
+	sticky: true,
+	shots: [
+		<GalleryTabDefault key="1" />,
+		<GalleryTabFlat key="2" />,
+		<GalleryTabHeight key="3" />,
+	],
+});
+
+export default meta;
+
+interface Props {
+	style?: string;
+	noPadding?: boolean;
+	fullHeight?: boolean;
+}
 
 const tabs: Tab[] = [
 	{ id: "first", title: "First", pane: () => <p>1st</p> },
 	{ id: "second", title: "Second", pane: () => <p>2nd</p> },
 ];
 
-export const Primary = (): JSX.Element => {
-	const [tab, setTab] = useState("first");
+export const Primary = (props: Props): JSX.Element => {
+	return (
+		<div>
+			<div style={{ height: "200px" }}>
+				<Tabs
+					children={tabs}
+					// eslint-disable-next-line
+					style={(Tabs.styles as any)[props.style!]}
+					noPadding={props.noPadding}
+					fullHeight={props.fullHeight}
+				/>
+			</div>
+		</div>
+	);
+};
+
+Utils.fixPrimary(Primary);
 
-	useEffect(() => {
-		console.log(`active tab is changed: ${tab}`);
-	}, [tab]);
+export const Basic = (): JSX.Element => {
+	const tabs: Tab[] = [
+		{ id: "first", title: "First", pane: () => <p>1st</p> },
+		{ id: "second", title: "Second", pane: () => <p>2nd</p> },
+	];
 
+	return <Tabs children={tabs} />;
+};
+
+Utils.desc(Basic)(`
+To begin, you need to provide an array of object with props: id, title and pane
+via children:
+`);
+
+export const With_Switcher = (): JSX.Element => {
+	const [tab, setTab] = useState("first");
 	return (
 		<div>
-			<Tabs children={tabs} />
-			<DivPx size={16} />
-			<Tabs children={tabs} noPadding />
-			<DivPx size={16} />
-			<Tabs style={Tabs.styles.flat} children={tabs} />
+			<Switcher<string>
+				value={tab}
+				setValue={setTab}
+				options={tabs.map((tab) => ({
+					value: tab.id,
+					label: tab.id,
+				}))}
+			/>
 			<DivPx size={16} />
-			<Tabs style={Tabs.styles.flat} children={tabs} noPadding />
-			<DivPx size={16} />
-			<div style={{ height: 200 }}>
-				<Tabs children={tabs} fullHeight />
-			</div>
-			<DivPx size={16} />
-			<div>
-				<p>
-					This is a controlled tabs. You can change the tab via these
-					buttons:
-				</p>
-				<DivPx size={16} />
-				<Switcher<string>
-					value={tab}
-					setValue={setTab}
-					options={tabs.map((tab) => ({
-						value: tab.id,
-						label: tab.id,
-					}))}
-				/>
-				<DivPx size={16} />
-				<Tabs children={tabs} setActiveTab={setTab} activeTab={tab} />
-			</div>
+			<Tabs children={tabs} setActiveTab={setTab} activeTab={tab} />
 		</div>
 	);
 };
+
+Utils.desc(With_Switcher)(`
+Tabs can also be use with
+[Switcher](https://docs.moaijs.com/?path=/docs/components-switcher--primary)
+to control tabs via buttons.
+`);
```

Comment:
This is ok to keep for now, but I think I'll change it later.. Using it with "Switcher" is just for the demo. The real topic that we want to show here is the "controlled" usage